### PR TITLE
Add $PROJECT_NAME variable support to simplify virtualenvwrapper support

### PR DESCRIPTION
**Pull request by Darune**
This is a simple way to handle virtualenvwrapper for me

### CODE REVIEW

File: lib/interpreters-lookup.coffee [hunk 1]
```diff
@@ -58,6 +58,8 @@ module.exports =
     for p in atom.config.get('autocomplete-python.pythonPaths').split(';')
       for project in atom.project.getPaths()
         modified = p.replace(/\$PROJECT/i, project)
+        project_folder_name = project.match(/([^\/]*)\/*$/)[1]
```
Comment (sadovnychyi) [inline lib/interpreters-lookup.coffee]: Should be something like `[_, ..., projectFolderName] = project.split(path.sep)` (and `require 'path'` somewhere if needed):
* Match coffeescript styleguide
* Correctly split path on windows where `\` is used
* Make it more obvious what we are doing here. Regex is hard to understand

**Comment by sadovnychyi** [comment]
I'll check everything and merge it next week. :+1:

File: lib/interpreters-lookup.coffee (hunk 1 above)
Comment (Darune) [inline lib/interpreters-lookup.coffee]: I'll fix that tonight and update the pull request

**Comment by sadovnychyi** [comment]
Released it in 1.5.0 with some changes. Thanks!
FYI I choose to fall back to `$PROJECT_NAME`.

**Pull request merged**